bindings/node/native/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "node-reconfix"
+version = "0.1.0"
+authors = ["Aaron Brodersen <aaron@resin.io>"]
+license = "MIT"
+build = "build.rs"
+
+[lib]
+name = "node_reconfix"
+crate-type = ["dylib"]
+
+[build-dependencies]
+neon-build = { git = "https://github.com/resin-io-modules/neon.git", branch = "fork" }
```

Comment:
I wonder if this branch should be given a more meaningful name than just "fork" ?  \*shrug\*

Reply:
@lurch The hope is that this branch will go away soon, and our changes will be mainlined.